test/IonTimestamp.ts

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
The `timestamp` tests are so heavily parameterized that I fell back to using conventional Mocha syntax (`describe`/`it`) instead of `mocha-typescript`'s declarative syntax (`@suite`/`@test`).

The end result is pretty nice and still benefits from Typescript strictness. For example, the JS version of this file had tests stored in an object keyed by the timestamp text to parse (`{timestampText: testToRun}`). JS allowed repeated keys, which meant that some tests were being overwritten and silently dropped. Typescript required me to change the data structure to preserve them.

As with the `@suite`/`@test` tests in the other files, Webstorm is able to run/debug individual tests/suites from within the browser.

Reply:
+1